packages/web/docs/src/app/blog/(posts)/introducing-mcp-graphql/page.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Introducing: GraphQL MCP server'
+authors: blurrah
+tags: [graphql, mcp]
+date: 2025-05-26
+description:
+  A new MCP server that connects AI models to GraphQL endpoints with full schema awareness and dynamic querying capabilities.
+---
+
+TL;DR
+
+- The GraphQL MCP server allows your AI models to interact with your GraphQL server
```

Comment:
Maybe worth mentioning in as many places as possible, that this works with ANY GraphQL server and you don't and shouldn't need any vendor specific solution in general